src/components/Checkout/SocialShare.vue

```diff
@@ -88,20 +97,29 @@ export default {
 	},
 	data() {
 		return {
-			selectedLoanIndex: 0,
+			copyStatus: {
+				class: '',
+				disabled: false,
+				text: 'Copy Link'
+			},
+			maxMessageLength: 280,
 			message: '',
-			maxMessageLength: 280
+			selectedLoanIndex: 0,
 		};
 	},
 	computed: {
+		selectedLoan() {
+			return this.loans[this.selectedLoanIndex];
+		},
+		selectedLoanUrl() {
+			return `https://www.kiva.org/lend/${this.selectedLoan.id}`;
```

Comment:
@emuvente This could be useful for the other social share pieces

Reply:
I think this needs to include the lender specific user name (unless anonymous)... is this the format that is currently used on the legacy thanks page?

Reply:
@mcstover yeah there are other things that need to be included for that whole share link. I'm adding that to my branch where I'm doing the other social share buttons. I can update the url the copy link uses with that so we can merge this pr